tests/admin_views/models.py

```diff
@@ -1147,6 +1147,7 @@ class Square(models.Model):
     area = models.GeneratedField(
         db_persist=True,
         expression=models.F("side") * models.F("side"),
+        output_field=models.IntegerField(),
```

Comment:
```suggestion
        output_field=models.BigIntegerField(),
```

Reply:
I don't see much value in changing to `BigIntegerField()` in tests.

Reply:
In the future contributors can expand these tests with integers big enough to not fit in the generated field.
As wrote before, many developers, myself included, read Django code, trusting that it is correct.  
What are your reasons for not making these changes?

Reply:
Already replied.